### PR TITLE
Task facade add

### PR DESCRIPTION
**Pull request by trevlenb2**

### CODE REVIEW

File: src/main/java/com/revealprecision/revealserver/RevealServerApplication.java [hunk 1]
```diff
@@ -15,6 +16,7 @@
 @EnableJpaAuditing
 @Slf4j
 @EnableConfigurationProperties
+@ServletComponentScan
```
Comment (lindagcaba) [inline src/main/java/com/revealprecision/revealserver/RevealServerApplication.java:19]: Is this being used with the current checkin? what are we scanning?

File: src/main/java/com/revealprecision/revealserver/api/v1/facade/models/TaskDto.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/**
```
Comment (lindagcaba) [inline src/main/java/com/revealprecision/revealserver/api/v1/facade/models/TaskDto.java:1]: we can delete these

File: src/main/java/com/revealprecision/revealserver/api/v1/facade/service/TaskFacadeService.java [hunk 3]
```diff
@@ -29,13 +50,30 @@
...
+  private final ActionService actionService;
+
+  private final PlanService planService;
+
+  private final PersonService personService;
+
+  private final LocationService locationService;
+
   @Autowired
   public TaskFacadeService(MetaDataJdbcService metaDataJdbcService, UserService userService,
```
Comment (lindagcaba) [inline src/main/java/com/revealprecision/revealserver/api/v1/facade/service/TaskFacadeService.java:64]: just a suggestion, if we introduce @RequiredArgsConstructor to class, we can get rid of this boilerplate code.(nothing wrong with this, but will result in less code)

File: src/main/java/com/revealprecision/revealserver/RevealServerApplication.java (hunk 1 above)
Comment (trevlenb2) [reply src/main/java/com/revealprecision/revealserver/RevealServerApplication.java:19]: This leaked in....it was meant to be with the fhir facade code....i had to redo this branch....

Il remove and ship this with the FHIR branch.....

File: src/main/java/com/revealprecision/revealserver/api/v1/facade/models/TaskDto.java (hunk 2 above)
Comment (trevlenb2) [reply src/main/java/com/revealprecision/revealserver/api/v1/facade/models/TaskDto.java:1]: thanks

File: src/main/java/com/revealprecision/revealserver/api/v1/facade/service/TaskFacadeService.java (hunk 3 above)
Comment (trevlenb2) [reply src/main/java/com/revealprecision/revealserver/api/v1/facade/service/TaskFacadeService.java:64]: thanks

**Review by lindagcaba — APPROVED** [review]

**Pull request merged**